adagrams/game.py

```diff
@@ -1,11 +1,136 @@
+import random
+import copy
+
 def draw_letters():
-    pass
+    
+    letter_frequency = {
+    'A': 9, 
```

Comment:
When wrapping a data structure literal like this, indent the contents one tab for readability (here down to line 33)

Reply:
By defining this data structure within the function, we get a fresh copy every time the function runs, which in certain situations may be important. We could move this data outside of the function (global in the module) and use an all caps variable to indicate to other developers that it should be treated as a constant. This would allow the function to be vertically shorter, which can help with readability.